<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: 18pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Iniciar Sesión</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputUser">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Usuario</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputPassword">
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Contraseña</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnLogin">
            <property name="text">
             <string>Ingresar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lblStatus">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="2">
           <layout class="QFormLayout" name="formLayout">
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="btnCreateUser">
                <property name="text">
                 <string>Crear Usuario</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnLogout">
                <property name="text">
                 <string>Cerrar Sesión</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lblWelcome">
              <property name="text">
               <string>Bienvenido al Sistema</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="txtSearch">
                <property name="placeholderText">
                 <string>Filtro</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnExport">
                <property name="text">
                 <string>Exportar a Excel/CSV</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableView" name="tableDevices">
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="3">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QPushButton" name="btnAddDevice">
              <property name="text">
               <string>Agregar Dispositivo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnEditDevice">
              <property name="text">
               <string>Editar Dispositivo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnDeleteDevice">
              <property name="text">
               <string>Eliminar Dispositivo</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>565</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcercaDe"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir...</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionAcercaDe">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
